### ISSUE TITLE
how can I generate a keyboard shortcut for supersave?

### ISSUE THREAD

**Issue by cornhundred** [maartenbreddels/jupyterlab-supersave#1]
The supersave extension mentioned [here](https://github.com/jupyterlab/jupyterlab/issues/5235) works for me, but I can't figure out how to make a keyboard shortcut. Do I need to add a JSON blob to the Keyboard Shortcuts settings?

**Comment by maartenbreddels** [comment]
Yeah indeed, I'm not sure if I can 'ship' a default for that, that would be nice.
